exchanges/bybit/bybit_wrapper.go

```diff
@@ -0,0 +1,1460 @@
+package bybit
```

Comment:
Now that things are starting to mature, I'd like to direct you to one of our helper applications, `exchange_wrapper_issues`. Its purpose is to highlight issues with our wrapper implementations.
If you run ` .\exchange_wrapper_issues.exe --exchanges="bybit"` you'll get an analysis of what's working and what isn't. It's not perfect and may have some inaccuracies, but for example, its highlighting that we're not formatting the pair in `GetRecentTrades` before calling the API, so it fails.

What I'd like from you is to add test coverage of wrapper functions. I know we don't always have it across our exchanges, but it really helps in highlighting potential assumptions when writing wrapper implementations. Functions such as `GetDefaultConfig()`, `SetDefaults()`, `Setup()`, `Start()`, `Run()` you don't need to worry about